.github/workflows/codeql-analysis.yml

```
@@ -0,0 +1,72 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "main", deploy-production ]
```

```suggestion
    branches: [ "main", production ]
```

It looks like `production` is the deploy branch in this repo, so we probably want to target this one?

---

oh yup good catch - I think it's fine to leave for now though since the majority of the same code should exist in `main`, so we can make the change the normal way.

---

👍 agreed, it just seemed like the kind of thing we would never loop back to if I didn't point it out